manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -27,14 +27,14 @@ meta:
     cloud_controller_ng: {}
 
   api_templates:
+  - name: consul_agent
+    release: (( grab meta.consul_templates.consul_agent.release ))
```

Comment:
Given the recommendation that this comes first, it's probably worth adding a unit test to ensure that whenever `consul_agent` is present in a job, it's listed first.